OpenRA.Mods.Common/Traits/SupportPowers/NukePower.cs

```diff
@@ -197,5 +218,72 @@ public void Activate(Actor self, WPos targetPosition)
 				});
 			}
 		}
+
+		public override void SelectTarget(Actor self, string order, SupportPowerManager manager)
+		{
+			Game.Sound.PlayToPlayer(SoundType.UI, manager.Self.Owner, Info.SelectTargetSound);
```

Comment:
Why was the `Game.Sound.PlayNotification` from the base `SupportPower.SelectTarget` removed?